spaceship/spec/tunes/app_details_spec.rb

```diff
@@ -35,8 +35,14 @@
       details.secondary_first_sub_category = "Weather"
       expect(details.secondary_first_sub_category).to eq("MZGenre.Weather")
 
-      details.secondary_second_sub_category = "Weather"
-      expect(details.secondary_second_sub_category).to eq("MZGenre.Weather")
+      details.secondary_second_sub_category = "Puzzle"
+      expect(details.secondary_second_sub_category).to eq("MZGenre.Puzzle")
+
+      details.primary_locale_code = "en-US"
```

Comment:
Sorry I'm not sure I'm following. Does this test relate to `it prefixes the category with the correct value for all category types`?
Also, just setting the value and retrieving it doesn't seem to actually test the code, instead we should test fetching the value from an existing application, right?